src/coreclr/scripts/genEventing.py

```diff
@@ -841,6 +1014,8 @@ def main(argv):
                                     help='if specified, will not generated extern function stub headers' )
     required.add_argument('--noxplatheader', action='store_true',
                                     help='if specified, will not write a generated cross-platform header' )
+    required.add_argument('--incList',  type=str,default="",
```

Comment:
nit: other args seem to do all lower case instead of camel case

Reply:
Also, genEventPipe.py have the same concept with include/exclude list for events, maybe we should make sure we use the same names for these arguments, (https://github.com/dotnet/runtime/blob/main/src/coreclr/scripts/genEventPipe.py#L1151), since --inc is already used in this script we should probably align on different name and use that for the same arguments.